.github/workflows/Deploy.yml

```
@@ -0,0 +1,92 @@
+name: "Deploy App"
+
+on:
+  push:
+    branches:
+            -master
+    paths-ignore:
+               .gitignore
+               README.MD
+
+jobs:
+  build-frontend:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '23'
+
+      - name: Build Frontend
+        working-directory: frontend
+        run: |
+          npm install
+          npm run build
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: frontend-build
+          path: frontend/dist/
+
+  build-backend:
+    runs-on: ubuntu-latest
+    needs: build-frontend
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: frontend-build
+          path: backend/src/main/resources/static
+
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: '23' # must match the version in the pom.xml
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Build with maven
+        run: mvn -B package --file backend/pom.xml
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: app.jar
+          path: backend/target/ApplicationManagementApp.jar # must match the finalName in the pom.xml
+
+  push-to-docker-hub:
+    runs-on: ubuntu-latest
+    needs: build-backend
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: app.jar
+          path: backend/target
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }} # must match the name of the Dockerhub account
```

```suggestion
          username: ${{ secrets.DOCKERHUB_USERNAME }}
```
these comments are superfluous, given that they are env variables